keps/sig-node/4622-topologymanager-max-allowable-numa-nodes/README.md

```diff
@@ -97,25 +99,48 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 
 In this KEP, we propose a new TopologyManager Policy Option called `max-allowable-numa-nodes` to configure the value of maxAllowableNUMANodes in the TopologyManager. The current hard-coded value of 8 was added as a stop-gap 4 years ago to mitigate the state explosion that occurs when trying to enumerate the possible NUMA affinities and generating their hints. By making this setting configurable, we give users the ability to increase this limit when appropriate.
 
-## Goals
+## Motivation
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
```

Comment:
if you resubmit, please remove comments like this from the completed sections.

Reply:
Thanks for your reminder.